installer/helm/chart/volcano/values.yaml

```diff
@@ -118,8 +118,11 @@ custom:
 #      cpu: 300m
 #      memory: 300Mi
   admission_resources: ~
+  admission_log_level: 4
```

Comment:
Maybe put these log reltaed together and append to the end of the file is better.

Reply:
ok, It's done